src/components/index.js

```diff
@@ -1,5 +1,7 @@
+import Search from './Search'
+
 const App = () => {
-  return <h2>HElloo</h2>
+  return <Search />
```

Comment:
You don`t need to use it now in App, because if I merge your pull-request that's create merge-conflict in another pull-request.